coderd/notifications/notifications_test.go

```diff
@@ -691,7 +691,7 @@ func TestNotificationTemplatesCanRender(t *testing.T) {
 					"reason":         "breached the template's threshold for inactivity",
 					"initiator":      "autobuild",
 					"dormancyHours":  "24",
-					"timeTilDormant": "24h",
+					"timeTilDormant": "24 hours",
```

Comment:
A test which includes not such a round number would also be helpful for illustrative purposes.

Reply:
Added